src/components/SearchForm.js

```diff
@@ -1,35 +1,37 @@
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
 
 const SearchForm = ({onFormChange}) => {
-  const [searchTerm, setSearchTerm] = useState("");
```

Comment:
By letting the form manage the data, we can actually make this component "stateless!"